### ISSUE TITLE
Create output logs

### ISSUE THREAD

**Issue by Gavriel94** [Gavriel94/ecoacoustics-pix2pix#42]
Each run of the models training should produce a new txt file stored in `cGAN/evaluation/timeanddate.txt` (`.txt` is debatable).

This should include
- The model's accuracy
- The loss values from all loss functions so graphs can be reproduced later
- The config of the models hyperparameters
- The number of samples in the training, validation and test set
